RoboFile.php

```diff
@@ -548,6 +548,20 @@ public function publish($opts = ['force' => false]) {
       ->run();
   }
 
+  function changelogUpdate($opts = ['version_name' => null]) {
+    $this->say("Updating changelog");
+    $outputs = $this->getChangelogController()->update($opts['version_name']);
+    if($opts['quiet']) {
+      return;
+    }
```

Comment:
What is this needed for? You already outputted something a few lines above.

Reply:
Since the repo is public also CIRCLE CI build outputs are accessible https://circleci.com/gh/mailpoet/mailpoet so this is here to hide the private notes we have for release on Circle